packages/core/src/components/meter/meter.scss

```diff
@@ -0,0 +1,86 @@
+:host {
+  align-items: flex-start;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  width: 100%;
+  word-wrap: break-word;
+}
+
+.container-text {
+  align-items: flex-end;
+  display: flex;
+  font: var(--text-body-small);
+  font-weight: 300;
+  justify-content: space-between;
+  letter-spacing: var(--text-body-small-letter);
+  margin-bottom: var(--spacing-xsmall);
+  width: 100%;
+
+  &.hasCenterTitle {
+    justify-content: center;
+  }
+
+  &.hasMarginTop {
+    margin-bottom: 0;
+    margin-top: var(--spacing-xsmall);
+  }
+
+  &.is-mobile {
```

Comment:
question: why do we create a new class instead of using breakpoints?

Reply:
I thought breakpoints would be better initially, but as I didn't see any use in the project, I chose to use classes haha, but using breakpoints is much better even in this case

Made: https://github.com/juntossomosmais/atomium/pull/695/commits/ce3e6d71f8999342744e3f3d0f97778ec5b7e8ec